install.sh

```diff
@@ -143,23 +143,23 @@ establish_firewall () {
 
 # Install CRI-O depending on the distro
 install_crio() {
+    CRIOVERSION=1.22
```

Comment:
1.22 isn't available in fc34, 1.21? 

And another question is:

1) why isn't it available in fc35,
2) what's the matching version expected to work with OKD 4.8 (what microshift is based on) (okd 4.8 seems to be based on K8s 1.21, but is the cri-o version aligned?)

Reply:
Hi @mangelajo I think there is something missing in fedora packages, because it's very strange that in the list I pointed in the issue, the 1.20 version is missing.
If the versions between kubectl and cri-o should be aligned, then maybe we can set the same version as the function `get_kubectl` downloads, how about it?

Reply:
get_kubectl is another topic, we should probably fix that one.

1.21 seems to be available in fc34 & 35, as well seems to be aligned with our openshift release. I propose we go for that one.

Reply:
```suggestion
    CRIOVERSION=1.21
```

Reply:
Sure @mangelajo, but if the cri-o version and kubectl version has to be the same (I don't know if when kubectl version is updated cri-o is updated too) we can get the cri-o package version number same as kubectl. I have prepared another function to do that in bash instead of when cri-o updates its version or deprecates it create a pull request changing the version. How about it?

Reply:
Openshift 4.9 uses crio 1.22 ... suggest we keep up ...